Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -353,6 +354,14 @@ void qSlicerCoreApplicationPrivate::init()
     vtkEventBroker::GetInstance()->SetRequestModifiedCallback(modifiedRequestCallback);
   }
 
+  // Create the translation object used for string translation in logic classes
+  {
+    if (!this->AppLogic->GetTranslatorInstance())
+    {
+      this->AppLogic->SetTranslatorInstance(new qMRMLTranslator());
+    }
+  }
+
```

Comment:
```suggestion
  if (!this->AppLogic->GetTranslatorInstance())
  {
    this->AppLogic->SetTranslatorInstance(new qMRMLTranslator());
  }
```

Reply:
There is no need for extra scope because there are no _stack allocated_[^1] object that need to be removed before reaching the end of the function. Above and below, we wanted the `vtkNew` and `QTemporaryFile` object to be "deleted".

[^1]: https://courses.engr.illinois.edu/cs225/fa2022/resources/stack-heap/